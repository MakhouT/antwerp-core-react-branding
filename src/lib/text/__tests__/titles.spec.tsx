import * as React from "react";

import * as A from "../..";
import { renderAndMatchSnapshot } from "../../../__tests__/__fixtures__/testUtils";

describe("Titles", () => {
    it("Renders H1", () => {
        // Act
        const element = (
            <A.H1>a title</A.H1>);

        // Act and Assert
        renderAndMatchSnapshot(element);
    });

    it("Renders H2", () => {
        // Act
        const element = (
            <A.H2>a title</A.H2>);

        // Act and Assert
        renderAndMatchSnapshot(element);
    });

    it("Renders H3", () => {
        // Act
        const element = (
            <A.H3>a title</A.H3>);

        // Act and Assert
        renderAndMatchSnapshot(element);
    });

    it("Renders H4", () => {
        // Act
        const element = (
            <A.H4>a title</A.H4>);

        // Act and Assert
        renderAndMatchSnapshot(element);
    });

    it("Renders H5", () => {
        // Act
        const element = (
            <A.H5>a title</A.H5>);

        // Act and Assert
        renderAndMatchSnapshot(element);
    });

});
