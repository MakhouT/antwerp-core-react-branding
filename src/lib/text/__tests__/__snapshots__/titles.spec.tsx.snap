// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titles Renders H1 1`] = `
<div
  className="u-margin-bottom-lg"
  style={undefined}
>
  <h1>
    a title
  </h1>
</div>
`;

exports[`Titles Renders H2 1`] = `
<div
  className="u-margin-bottom"
  style={undefined}
>
  <h2>
    a title
  </h2>
</div>
`;

exports[`Titles Renders H3 1`] = `
<div
  className="u-margin-bottom"
  style={undefined}
>
  <h3>
    a title
  </h3>
</div>
`;

exports[`Titles Renders H4 1`] = `
<div
  className="u-margin-bottom-xs"
  style={undefined}
>
  <h4>
    a title
  </h4>
</div>
`;

exports[`Titles Renders H5 1`] = `
<div
  className="u-margin-bottom-xx"
  style={undefined}
>
  <h5>
    a title
  </h5>
</div>
`;
